src/index.html

```diff
@@ -6,13 +6,62 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
     <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
       rel="stylesheet"
-      href="./styles/index.scss"
     />
+    <link
+      rel="stylesheet"
+      href="./main.js"
+    />
+    <title>Product cards</title>
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="./images/imac.jpeg"
+        class="card__img"
+        alt="product-img"
+      />
+      <h3 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+      <div class="card__code">Product code: 195434</div>
+      <div class="card__ranking">
+        <div class="card__stars">
+          <ul class="stars">
+            <li class="stars__star star-active"></li>
+            <li class="stars__star star-active"></li>
+            <li class="stars__star star-active"></li>
+            <li class="stars__star star-active"></li>
+            <li class="stars__star star-active"></li>
+          </ul>
+        </div>
+        <p class="card__reviews">Reviews: 5</p>
+      </div>
+      <div class="card__price">
+        <p class="card__price-title">Price:</p>
+        <p class="card__price-value">$2,199</p>
+      </div>
+      <a
+        href="#checkout"
+        class="card__link"
+        data-qa="hover"
+      >
+        Buy
+      </a>
```

Comment:
The 'Buy' button should be a 'button' or 'input' element with 'type="submit"', not a link. Links are used to navigate to different pages or sections within a page, not to submit forms.